Code/Modules/player/psp/pspScenePlayer.cpp

```diff
@@ -222,15 +223,15 @@ namespace {
 
 //;;printf(" blastSurfaceInputs -- toNativeColors\n");
 
-//		printf("�� src.diffuseTexture = %x\n", src.diffuseTexture);
+//		printf("�� src.diffuseTexture = %x\n", src.diffuseTexture);
 
 		dst.diffuseTexture = 0;
 		dst.envmapTexture = 0;
 		if(src.diffuseTexture != ~0U)
 			dst.diffuseTexture = &scene.mResources.textures[src.diffuseTexture].renderable->mBlueprint;
 		if(src.envmapTexture != ~0U)
 			dst.envmapTexture = &scene.mResources.textures[src.envmapTexture].renderable->mBlueprint;
-//		printf("�� dst.diffuseTexture = %x\n", dst.diffuseTexture);
+//		printf("�� dst.diffuseTexture = %x\n", dst.diffuseTexture);
```

Comment:
GitHub displays this differently than the other random symbols in `printf`. This seems to be unintentional.